### PR TITLE
example pull request from Slavik

### PR DESCRIPTION
**Pull request by viatk**
Hi Kshis!

### CODE REVIEW

File: src/Plot_GC_Bias.R [hunk 1]
```diff
@@ -77,4 +77,6 @@ jpeg(out_file_name,width=300, height=300)
   text(0.1,5.5,pval)
 dev.off()
 }
-
```
Comment (kshis) [inline src/Plot_GC_Bias.R:80]: Remove blank line

Comment (viatk) [reply src/Plot_GC_Bias.R:80]: ok

File: src/Plot_GC_Bias.R [hunk 2]
```diff
@@ -76,5 +76,6 @@ jpeg(out_file_name,width=300, height=300)
        lty = c(1,2), col = c("red","green"))
   text(0.1,5.5,pval)
 dev.off()
-}
```
Comment (kshis) [inline src/Plot_GC_Bias.R:79]: ttt

Comment (viatk) [reply src/Plot_GC_Bias.R:79]: aaa

**Comment by kshis** [comment]
bu

**Pull request closed without merging**